project-reports/2023/2023-Q3-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped, and there are no plans for major releases.
+The release `2.0.0-pre-rc.18` is officially merged and stable as of July 31th.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+The integration of Ursa essential parts to the Iroha codebase continues.
+
+# Questions/Issues for the TSC
+
+- No indexing on [2023](https://wiki.hyperledger.org/display/TSC/2023+Project+Updates) page
+- LFX interface [caused problems](https://community.lfx.dev/t/lfx-mentorship-platform-issues/2130) for candidate vetting/selection, and internship allocations
+
+# Releases
+
+Past quarter releases:
+* v2.0.0-pre-rc.15
+* v2.0.0-pre-rc.16
+* v2.0.0-pre-rc.17
+* v2.0.0-pre-rc.18
+
+Iroha 1 is in maintenance mode. No releases are planned until the completion of the internship project.
+
+# Overall Activity in the Past Quarter
+
+According to the [Iroha Activity Dashboard](https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Firoha/dashboard;subTab=technical?time=%7B%22from%22:%222023-04-01T06:38:18.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-07-31T21:00:00.000Z%22%7D) for the third quarter of 2023 (April-June), the Iroha repo has 228 PRs submitted by 44 contributors.
```

Comment:
This is great. There is increase in contributors count, though most of them are reported as individuals interested in the project.

Reply:
Thanks for pointing that out; I'll DM you to discuss the details because I need clarification about the LFX interface.